app.js

```diff
@@ -2,44 +2,33 @@ angular.module('PathOfDamage', [])
 .controller('Damage', function ($scope, DataService) {
   $scope.sections = DataService.getSections();
 
-  $scope.hits = [{hit: 100}, {hit: 500}, {hit: 1000}, {hit: 2000}, {hit: 3000},
-    {hit: 4000}, {hit: 5000}, {hit: 7500}, {hit: 10000}];
+  $scope.hits = [{hit: 100}, {hit: 500}, {hit: 1000}, {hit: 2000}, {hit: 3000}, {hit: 5000}, {hit: 7500}, {hit: 10000}];
   $scope.resistance = 75;
-  $scope.healthPool = 7000;
+  $scope.healthPool = 5000;
```

Comment:
RIP ES
Do you think 5k is actually the average life+es pool people have? Or do you expect only those left-side-of-the-tree only-use-life types to visit the site?